content/en/docs/reference/configuration/registration/_index.md

```diff
@@ -86,6 +84,29 @@ when it is required.**
 `external-auth-attribue` must be the name of the IdP attribute, or token claim (when provided by SAML IdPs,
 or OIDC Providers, respectively) to use for the mentioned account creation form field.
 
+
+## Automatic enrollment trough SAML IdPs
+
+In case of registration trough an external SAML Identity Provider, IAM offers
+a flexible user enrollment flow, also without IAM admin intervention. The default IAM
+behavior is that the user enrollment requires an administrator approval step.
+
+In order to enable an automatic enrollment flow trough an external IdP, one
```

Comment:
In order to enable **the** automatic enrollment flow **via** an external IdP, ...